db/migrate/011_add_disabled_to_deliverables.rb

```diff
@@ -0,0 +1,5 @@
+class AddHiddenToDeliverables < ActiveRecord::Migration
+  def change
+    add_column :hidden, :disabled, :boolean, :default => false
```

Comment:
The "disabled" column seems to be unused and the filename does not match the migration (class) name.